packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -349,6 +351,16 @@ class GoogleMap extends StatefulWidget {
   /// See [WebGestureHandling] for more details.
   final WebGestureHandling? webGestureHandling;
 
+  /// This setting controls how the API handles cameraControl button position on the map. Web only.
+  ///
+  /// See [WebCameraControlPosition] for more details.
+  final WebCameraControlPosition? webCameraControlPosition;
+
+  /// This setting controls how the API handles cameraControl button on the map. Web only.
```

Comment:
Have a more descriptive comment here. Something like "Enables or disables the Camera controls."

Reply:
done.